static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -0,0 +1,45 @@
+# Caveats running DVC on Windows
+
+There can be few performance issues while running DVC on Windows by default. For
+some of these issues there are workarounds or configuration options available.
+Below are some of these issues detected so far and their workarounds. If you
+find any, please let us know!
+
+DVC in general is slower on Windows as compared to other platforms (like
+Linux, MacOS, etc.) that show pretty good performance. Upon investigation, it
+turns out that most of the delays are coming from [NTFS](https://en.wikipedia.org/wiki/NTFS)
+(New Technology File System) not being very good at handling directories with
+large number of files. NTFS is a file system that the Windows NT operating
+system uses for storing and retrieving files on a hard disk. [Here](https://superuser.com/questions/15192/bad-ntfs-performance)
```

Comment:
have you run the pre-commit hook? the link should go on the next line most likely

Reply:
No but both the checks passed. If I'll put some links on next line, even then they will be more than 80 characters. I think lint checks should ignore links.

Reply:
CI does not check the style. Please, run the prettier manually, and install the pre-commit hook to keep the style the same.

Reply:
Done